tests/test_series.py

```diff
@@ -1212,7 +1228,10 @@ def add1(x: int) -> int:
     check(assert_type(s5, "pd.Series[int]"), pd.Series, np.integer)
     # inplace
     # TODO fix issue with inplace=True returning a Series, cf pandas #60942
-    s6: None = pd.Series([1, 2, 3]).rename("A", inplace=True)
+    check(
+        assert_type(pd.Series([1, 2, 3]).rename("A", inplace=True), "pd.Series[int]"),
+        pd.Series,
+    )
```

Comment:
You should add a test for specifying `inplace=True` with `index=None` being explicit, as well as a test for specifying `inplace=True` with `index` being a dict.